docs/reference/manifests.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: 'Gadget Instance Manifests'
+sidebar_position: 305
+description: Running gadgets in a predefined way using manifests
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Instead of providing all the parameters from command line, you can also use the `-f` flag in combination with the `run`
+command to specify a manifest file to run gadgets from. A manifest file is a file in
+[YAML format](https://en.wikipedia.org/wiki/YAML), that can hold information about one or more gadget instances. Such a
+file could look like this:
+
+```yaml
+apiVersion: 1
+kind: instance-spec
+image: trace_exec
+name: my-gadget-1
+paramValues:
+  operator.LocalManager.host: true
+tags:
+  - mytag1
+  - mytag2
+---
+apiVersion: 1
+kind: instance-spec
+image: trace_open
+name: my-gadget-2
+```
+
+This example contains two gadget instance specifications that are divided by three dashes (`---`).
+
+The `apiVersion` and `kind` fields define the specification used. They are mandatory and currently we only support
+this version and kind as part of the manifest.
+
+Specifying an `image` is also mandatory, while the rest of the fields are optional.
+
+When specifying `paramValues`, please use the fully qualified parameter names provided with their respective
+documentations in the [operators section](../spec/operators) ([example](../spec/operators/filter#filter)).
+
+:::warning
+
+If a manifest contains multiple instance specs, it can only be run by additionally using `--detach`. This is however
+only available when using `gadgetctl`. The run command will then try to create instances for all given specs and return
+either their IDs.
```

Comment:
```suggestion
their IDs.
```

Or what's the role of "either" in his sentence?

Reply:
It was "either an error or..." at one point, but I decided against using it. Didn't catch that when re-reading 😄